tests/integration/test_package.py

```diff
@@ -52,13 +52,13 @@ def _get_random_package_name():
                     reason='Extended packaging tests only run on py3.')
 @pytest.mark.parametrize(
     'package,contents', [
-        ('pandas==1.1.5', [
+        ('pandas==1.5.3' if sys.version_info[1] >= 10 else 'pandas==1.1.5', [
```

Comment:
I had to update a lot of the package versions to ones that would work with 3.10.  I tried to find releases that would work for 3.7 up to 3.10, but wasn't able to for pandas, so split it out to use a different version for 3.10 and up.

Reply:
Sounds good.  Given this is already run as part of the slower CI test runs, I might split these out into different versions per python version just to ensure we don't regress on supporting these packages for older versions of python.

Reply:
Pushed https://github.com/aws/chalice/pull/2036/commits/c7906bfc34126a3cf6599601ade299ce6ec1afe5 for what I had in mind.  I'm being cautious here, but I'm preserving the existing package version for python < 3.10, and then all python versions >= 3.10 will use the versions you have in your PR.

Assuming the CI passes, I'll go ahead and merge, unless there's any concerns with this approach.  Thanks again!

Reply:
thanks James!